### ISSUE TITLE
Failed to get metadata

### ISSUE THREAD

**Issue by berto1989** [iammert/RadioPlayerService#19]
Hello, I'm using your RadioPlayerService but when retrieve the metadata does not return me the artist and the song, I saw that the stream returns me:

D/IcyInputStream: Metadata string: StreamTitle='Kotoko - Loop-the-Loop';StreamUrl='&artist=UNISON%20SQUARE%20GARDEN&title=&album=&duration=254067&songtype=S&overlay=no&buycd=&website=&picture=az%5FB3102%5F%5FUNISON%20SQUARE%20GARDEN%2Ejpg';

But in the sample it has "singname" etc and I do not put it in the notification

How could fix this?

Ty for your attention

**Comment by iammert** [comment]
Seems like "StreamTitle='Kotoko - Loop-the-Loop " is your singer and song name. Am I wrong? you can parse it.

**Comment by berto1989** [comment]
Yes, but the notification does not write the song, I want to get the song in a variable like the artist, how can I do that?
